src/modules/72-triggers/pages/triggers/views/SchedulePanel.tsx

```diff
@@ -23,15 +24,21 @@ const SchedulePanel: React.FC<SchedulePanelPropsInterface> = ({
     values
   },
   formikProps,
-  isEdit = false
+  isEdit = false,
+  isFixedUTCTime
```

Comment:
Will make absolutely no difference to the code, but it's probably a good practice to default this to `false`, otherwise it could be a `boolean` or `undefined`. Total nitpick since the code will treat `false` and `undefined` the same.
```suggestion
  isFixedUTCTime = false
```